src/bindings/python/src/openvino/__init__.py

```diff
@@ -57,6 +57,6 @@
 # Tools
 try:
     # Model Conversion API - ovc should reside in the main namespace
-    from openvino.tools.ovc import convert_model
+    from openvino.tools.ovc import convert_model, InputCutInfo
```

Comment:
```suggestion
    from openvino.tools.ovc import convert_model
```
`InputCutInfo` should be removed in all places if import is coming from `openvino.tools.ovc`. If for some reason it was left somewhere, it should be removed.
Ref: https://github.com/openvinotoolkit/openvino/pull/18927

Reply:
Done.